packages/ember-data/lib/system/store.js

```diff
@@ -440,9 +440,32 @@ Store = Ember.Object.extend({
     store.find('person', { page: 1 });
     ```
 
-    This will ask the adapter's `findQuery` method to find the records for
-    the query, and return a promise that will be resolved once the server
-    responds.
+    This will call the private method `findQuery`, which delegates the query to the adapter. The adapter then makes a call to the server, which will return a `RecordArray` when the promise resolves.
+
+    Exposing queries this way seems preferable to creating an abstract query
+    language for all server-side queries, and then require all adapters to
+    implement them.
+
+    The call made to the server, depending on what backend you are using, will look something like this:
+
+    ```
+    Started GET "/api/v1/person?page=1"
+    Processing by Api::V1::PersonsController#index as HTML
+    Parameters: {"page"=>"1"}
```

Comment:
Sure thing. I wasn't sure if it would be good to include a specific backend framework rather than trying to be agnostic, but I wanted to provide an example to aid in the clarification.